internal/envoy/v3/route.go

```diff
@@ -117,6 +117,39 @@ func RouteDirectResponse(response *dag.DirectResponse) *envoy_route_v3.Route_Dir
 	}
 }
 
+// RouteRedirect creates a *envoy_route_v3.Route_Redirect for the
+// redirect specified. This allows a redirect to be returned to the
+// client.
+func RouteRedirect(redirect *dag.Redirect) *envoy_route_v3.Route_Redirect {
```

Comment:
nit: do we usually add units for these small helpers? could be nice to avoid future mistakes in case of refactor

Reply:
:+1: added.